extensions/common/gcp/gcp-core/src/test/java/org/eclipse/edc/gcp/bigquery/PartAssert.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2024 Google LLC
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Google LLC
+ *
+ */
+
+package org.eclipse.edc.gcp.bigquery;
+
+import org.assertj.core.api.AbstractAssert;
+import org.eclipse.edc.connector.dataplane.spi.pipeline.DataSource.Part;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PartAssert extends AbstractAssert<PartAssert, Part> {
```

Comment:
doc. and does this have to be `public`? seems like package-private would be fine too.